cypress/integration/userCanSearchDrinks.feature.js

```diff
@@ -0,0 +1,38 @@
+describe("User can search", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("http://localhost:3001");
+  });
+  it("search field exists", () => {
+    cy.get("#name-search.prompt").should("exist");
+    cy.get("#search").should("exist");
+    cy.get("#search").contains("Search");
+  });
+  it("successfully by drink name", () => {
+    cy.route({
+      method: "POST",
+      // url: "http://localhost:3000/api/v1/cocktails",
+      url: "https://cocktails-api-team1.herokuapp.com/api/v1/cocktails", //need to update later
+      response: "fixture:margarita_drink_search.json"
+    });
+    cy.get("#name-search.prompt").type("Margarita");
+    cy.get("button")
+      .contains("Search")
+      .click();
+    cy.get("#result-list").should("contain", "Margarita");
+  });
+
+  it("unsuccessfully by drink name", () => {
+    cy.route({
+      method: "POST",
+      // url: "http://localhost:3000/api/v1/cocktails",
+      url: "https://cocktails-api-team1.herokuapp.com/api/v1/cocktails", //need to update later
+      response: { status: 400, message: ["No drinks were found"] }
```

Comment:
Why is the message an array? Is that what your back-end will return, and if so, why is that?

Reply:
Using the console with debugger we thought this was an array, we have realized now that it is not coming from the backend that way.